[2024-02-10T02:08:19.690+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-02-03T05:00:00+00:00 [queued]>
[2024-02-10T02:08:19.700+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-02-03T05:00:00+00:00 [queued]>
[2024-02-10T02:08:19.700+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-02-10T02:08:19.723+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2024-02-03 05:00:00+00:00
[2024-02-10T02:08:19.731+0000] {standard_task_runner.py:60} INFO - Started process 206 to run task
[2024-02-10T02:08:19.737+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'load_data_to_postgres', 'scheduled__2024-02-03T05:00:00+00:00', '--job-id', '206', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmp924o72lo']
[2024-02-10T02:08:19.739+0000] {standard_task_runner.py:88} INFO - Job 206: Subtask load_data_to_postgres
[2024-02-10T02:08:19.811+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-02-03T05:00:00+00:00 [running]> on host 324edfad8c3f
[2024-02-10T02:08:20.008+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-02-03T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-03T05:00:00+00:00'
[2024-02-10T02:08:20.035+0000] {logging_mixin.py:188} INFO - /opt/***/dags/weather_data2024-02-03T05:00:00+00:00.json
[2024-02-10T02:08:20.035+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-10T02:08:20.050+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=load_data_to_postgres, execution_date=20240203T050000, start_date=20240210T020819, end_date=20240210T020820
[2024-02-10T02:08:20.129+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-10T02:08:20.198+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
